src/controller/WriteInteraction.h

```diff
@@ -50,8 +50,8 @@ class WriteCallback final : public app::WriteClient::Callback
     using OnErrorCallbackType = std::function<void(const app::ConcreteAttributePath * path, CHIP_ERROR err)>;
     using OnDoneCallbackType  = std::function<void(app::WriteClient *)>;
 
-    WriteCallback(OnSuccessCallbackType aOnSuccess, OnErrorCallbackType aOnError, OnDoneCallbackType aOnDone) :
-        mOnSuccess(aOnSuccess), mOnError(aOnError), mOnDone(aOnDone), mCallback(this)
+    WriteCallback(OnSuccessCallbackType aOnSuccess, OnErrorCallbackType aOnError, OnDoneCallbackType aOnDone, bool aIsGroupWrite) :
```

Comment:
should we name this along the line `requireResponse` or similar rather than specifying groups? Maybe that is less coupling

Reply:
This whole file is pretty coupled...  I would rather leave this as-is so it's clear which cases we are in fact considering.